apps/web/next.config.js

```diff
@@ -14,6 +14,11 @@ module.exports = withTM({
     });
     return config;
   },
+  experimental: {
```

Comment:
I don't think you need to add, we are using next 12.3.0, which already has a stable version of this component.

Reply:
removed